core/src/utilities/constants.js

```diff
@@ -0,0 +1,6 @@
+export const topNavDefaults = {
```

Comment:
I suggest uppercase + `_` notation for global constants

Reply:
And I agree with you, so I will change it. Thanks, I missed that one